src/librustdoc/test.rs

```diff
@@ -924,4 +932,19 @@ assert_eq!(2+2, 4);".to_string();
         let output = make_test(input, None, true, &opts);
         assert_eq!(output, (expected.clone(), 1));
     }
+
+    #[test]
+    fn make_test_display_warnings() {
+        //if the user is asking to display doctest warnings, suppress the default allow(unused)
+        let mut opts = TestOptions::default();
+        opts.display_warnings = true;
+        let input =
+"assert_eq!(2+2, 4);";
```

Comment:
Why this strange indent?

Reply:
Since `make_test` doesn't add indentation to any of the code it parses, i moved everything to the far left margin so i wouldn't have to deal with other ways of keeping the indentation right. For these one-line examples it's not necessary but i wanted both the "before" and "after" to be on the same indent level to visually compare them more easily.